<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brother&apos;s Gym Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="taekwondoButton">
      <property name="text">
       <string>Taekwondo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="womenButton">
      <property name="text">
       <string>Cours Femme</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="musculationButton">
      <property name="text">
       <string>Musculation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addCourseButton">
      <property name="text">
       <string>Ajouter cours</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitButton">
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
